vehicle-model-lifecycle/test/unit/test_download_vspec.py

```diff
@@ -137,10 +137,11 @@ def test_main__no_vehicle_signal_interface__adds_default_to_cache():
     with capture_stdout() as capture, mock_env():
         main(app_manifest)
 
-        expected_path = str(
-            Path(__file__).parent.parent.joinpath(
-                "vehicle-model-lifecycle", "vspec.json"
-            )
-        )
+        expected_path = str(os.path.join(get_project_cache_dir(), "vspec.json"))
+        expected_unit_path = str(os.path.join(get_project_cache_dir(), "units.yaml"))
```

Comment:
shouldn't this be the package_dir instead of cache_dir, since the default is in the root of this package?
the file is only in the cache_dir if it is specified as uri

Reply:
you're right interesting that the test ran through